lightning/src/util/config.rs

```diff
@@ -151,6 +156,7 @@ impl Default for ChannelHandshakeLimits {
 	fn default() -> Self {
 		ChannelHandshakeLimits {
 			min_funding_satoshis: 0,
+			max_funding_satoshis: 1 << 24,
```

Comment:
IMO it's unimportant, just a reasonable default.. If there's a strong preference for using the constant I can switch

Reply:
Figure if we change the constant to `(1 << 24) - 1` and use `>` instead of `>=` in the checks, then the "max" naming of this field and the constant would be accurate. Currently, they are off by one. Then might as well use the constant (or another with the same value) here rather than have the new default be one more than the old default, which would occur if the above recommended changes are made.